src/import/export.c

```diff
@@ -119,8 +119,7 @@ static int export_tar(int argc, char *argv[], void *userdata) {
         if (r < 0)
                 return log_error_errno(r, "Failed to run event loop: %m");
 
-        log_info("Exiting.");
-        return -r;
+        return log_info_errno(SYNTHETIC_ERRNO(r), "Exiting.");
```

Comment:
Hmm, could you please drop this chunk from this patch? This looks very iffy, because r >= 0 at this point. It might need some cleanup in a separate step.